BSP/Simulator_V2/Src/BSP_Fans.c

```diff
@@ -0,0 +1,65 @@
+#include "BSP_Fans.h"
+#include "Simulator.h"
+
+#define NUM_FANS 4
```

Comment:
Should move this to config.h so we don't have to change for both BSP layers if we do change.

Reply:
Will do